modules/misc/news/2025-04-26_18-56-13.nix

```diff
@@ -0,0 +1,11 @@
+{ pkgs, config, ... }:
+{
+  time = "2025-04-26T13:26:13+00:00";
+  condition = pkgs.hostPlatform.isLinux && config.services.espanso.enable;
```

Comment:
`pkgs.hostPlatform` is an alias. https://github.com/nix-community/home-manager/pull/6931

Reply:
@zowoq for my learning, why is using an alias a problem here? Actually, NM -- looks like that's made pretty clear [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/aliases.nix). Did you pick this up just from memory? Or is there a linter or something that might help me notice this going forward?

Reply:
nixpkgs has an [allowAliases](https://github.com/NixOS/nixpkgs/blob/88b87b7a62ef7c1c70510097f01f656a428d293e/pkgs/top-level/config.nix#L86) option.

Reply:
Thanks!